<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>logManager</class>
 <widget class="QWidget" name="logManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1540</width>
    <height>735</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>思源黑体 CN Light</family>
    <pointsize>-1</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-family: &quot;思源黑体 CN Light&quot;;
font-size: 32px;
font-weight: bold;
line-height: 49px;
letter-spacing: 1px;
color: 0
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>起始止时间</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateTimeEdit_Start">
         <property name="minimumSize">
          <size>
           <width>340</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>思源黑体 CN Light</family>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="dateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2021</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
         <property name="displayFormat">
          <string>yyyy/MM/dd HH:mm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>3</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>3</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateTimeEdit_Stop">
         <property name="minimumSize">
          <size>
           <width>340</width>
           <height>60</height>
          </size>
         </property>
         <property name="dateTime">
          <datetime>
           <hour>23</hour>
           <minute>59</minute>
           <second>0</second>
           <year>2021</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
         <property name="date">
          <date>
           <year>2021</year>
           <month>1</month>
           <day>1</day>
          </date>
         </property>
         <property name="displayFormat">
          <string>yyyy/MM/dd HH:mm</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_Search">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>48</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>240</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>查询日志</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Export">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>48</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>240</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>导出日志</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Del">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>48</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>240</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>删除日志</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>序号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>文件名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>起始时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>文件大小</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
